modules/core/assets/i18n/english/languagemenu_en.lang

```diff
@@ -0,0 +1,5 @@
+{
+  "english" : "English",
+  "german" : "Deutsch",
```

Comment:
In Terasology we actually have _two_ fields for languages - their name in English and their name in their origin language. Maybe we should have something like that in DS as well?

Reply:
Sure, which variation should appear in the language menu though?

Reply:
What about both? `"Deutsch (German)"`